hnn_core/network_builder.py

```diff
@@ -470,6 +489,12 @@ def _connect_celltypes(self):
                             target_cell.synapses[syn_key])
                         self.ncs[connection_name].append(nc)
 
+    def _record_extracellular(self):
+        for arr_name, arr in self.net.rec_array.items():
+            nrn_arr = ExtracellularArrayBuilder(arr)
+            nrn_arr._build(cvode=_CVODE)
+            self._rec_array.update({arr_name: nrn_arr})
+
```

Comment:
Nitpick: I think this is only used once so maybe we can move these lines to `_build()`?

Reply:
I do see you point (and agree), but I think it fits with the general scheme of keeping `_build` procedural? I'm going to leave this here for now. A future PR can improve on `_build`.

The call to `nrn_arr._build` is a little superfluous; should it remain or be included in `ExtracellularArrayBuilder.__init__`?

Reply:
I say keep it here if we sticking to the rule of ultra-lightweight `__init__`'s